docs/tutorial.md

```diff
@@ -0,0 +1,351 @@
+# Stackdriver Sandbox User Guide
+
+## Overview
+
+The Stackdriver Sandbox is intended to make it easy for you to deploy and run a non-trivial application that lets you explore the Google Cloud Platform services, particularly the [Ops Management](http://cloud.google.com/products/operations) (formerly Stackdriver) product suite. Ops Management is a suite of tools that helps you gain full observability into your code and applications.
```

Comment:
Since we need to rebrand to "Operations Management Sandbox' in the upcoming release, all new docs should replace "Stackdriver" with "Operations Management" when we refer to a group of products together.

Reply:
Okay. In my next commit I fix everything that says "Stackdriver" except for the URLs and example terminal output.